ui/lib/core/addon/templates/components/replication-secondary-card.hbs

```diff
@@ -15,18 +15,23 @@
       {{/if}}
     </div>
     <div class="grid-item-right">
-      {{#if (or (eq connection "transient-failure") (eq connection "shutdown"))}}
+      {{#if (eq connection "transient-failure")}}
```

Comment:
it looks like this is actually called`transient_failure`:

![image](https://user-images.githubusercontent.com/903288/79809039-5963de80-8324-11ea-8913-5cb322f4cfb0.png)